akka-actor/src/main/scala/akka/actor/Props.scala

```diff
@@ -156,10 +112,9 @@ object Props {
  * }}}
  */
 @SerialVersionUID(2L)
-final case class Props(deploy: Deploy, clazz: Class[_], args: immutable.Seq[Any]) {
+final case class Props(deploy: Deploy = Props.defaultDeploy, clazz: Class[_], args: immutable.Seq[Any]) {
```

Comment:
Actually this is no more needed, so I remove this and put the modification into ```AbstractProps```.
will see this into the next commit.